File: android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/ImageDisplayActivity.java
package com.liferay.mobile.screens.testapp;

import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ImageView;
import android.widget.Spinner;
import com.liferay.mobile.screens.asset.AssetEntry;
import com.liferay.mobile.screens.asset.display.AssetDisplayListener;
import com.liferay.mobile.screens.dlfile.display.image.ImageDisplayScreenlet;

/**
 * @author Sarai Díaz García
 */
public class ImageDisplayActivity extends ThemeActivity
    implements AssetDisplayListener, AdapterView.OnItemSelectedListener {

    private ImageDisplayScreenlet screenlet;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.image_display);

        screenlet = findViewById(R.id.image_display_screenlet);
        screenlet.load();
        screenlet.setListener(this);

        Spinner spinner = findViewById(R.id.spinner_scale_type);
        spinner.setSelection(3);
        spinner.setOnItemSelectedListener(this);
    }

    @Override
    public void error(Exception e, String userAction) {
        error(getString(R.string.asset_error), e);
    }

    @Override
    public void onRetrieveAssetSuccess(AssetEntry assetEntry) {
        info(getString(R.string.asset_received_info) + " " + assetEntry.getTitle());
    }

    @Override
    public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
        screenlet.setScaleType(ImageView.ScaleType.values()[position]);
    }

    @Override
    public void onNothingSelected(AdapterView<?> parent) {
    }
}

File: xamarin/Samples/Showcase-Android/Activities/ImageDisplayActivity.cs
using System;
using Android.App;
using Android.OS;
using Android.Widget;
using Com.Liferay.Mobile.Screens.Asset;
using Com.Liferay.Mobile.Screens.Asset.Display;
using Com.Liferay.Mobile.Screens.Dlfile.Display.Image;
using Java.Lang;

namespace ShowcaseAndroid
{
    [Activity]
    public class ImageDisplayActivity : Activity, IAssetDisplayListener
    {
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);
            SetContentView(Resource.Layout.ImageDisplayView);

            ImageDisplayScreenlet imageDisplayScreenlet =
                (ImageDisplayScreenlet) FindViewById(Resource.Id.image_display_screenlet);
            imageDisplayScreenlet.Listener = this;

            imageDisplayScreenlet.Load();
        }

        /* IAssetDisplayListener */

        public void OnRetrieveAssetSuccess(AssetEntry p0)
        {
            Toast.MakeText(this, "Image display success: " + p0.EntryId, ToastLength.Short).Show();
        }

        public void Error(Java.Lang.Exception p0, string p1)
        {
            System.Diagnostics.Debug.WriteLine($"Image display failed: {p0.Message}");
        }
    }
}
